docs/root/intro/version_history.rst

```diff
@@ -100,6 +100,10 @@ Version history
 * stats: added unit support to histogram.
 * tcp_proxy: the default :ref:`idle_timeout
   <envoy_api_field_config.filter.network.tcp_proxy.v2.TcpProxy.idle_timeout>` is now 1 hour.
+* tcp_proxy: added :ref:`hash_policy<envoy_api_field_config.filter.network.tcp_proxy.v2.TcpProxy.hash_policy>`
```

Comment:
Please merge master and move to 1.13 section.

Reply:
Updated.